variables.tf

```diff
@@ -6,12 +6,12 @@ variable "namespace" {
 variable "repositories" {
   description = "A list of repository defintions"
   default     = []
-  type        = list(map(string))
+  type        = list(string)
 }
 
 variable "chart_version" {
   description = "version of charts"
-  default     = null
+  default     = ">=2.3.3"
```

Comment:
The chart version is `4.5.10` https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/Chart.yaml#L5. `2.3.3` is the `appVersion`

Reply:
OK